content/en/blog/_posts/XXXX-XX-XX-hpa-configurable-tolerance.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: "Kubernetes v1.33 and Horizontal Autoscaling Configurable Tolerance"
+slug: hpa-configurable-tolerance
+# after the v1.33 release, set a future publication date and remove the draft marker
+# the release comms team can confirm which date has been assigned
+#
+# PRs to remove the draft marker should be opened BEFORE release day
+draft: true
+math: true # for formulae
+date: XXXX-XX-XX
+author: "Jean-Marc François (Google)"
+---
+
+This post describes _configurable tolerance for horizontal Pod autoscaling_,
+a new alpha feature first available in Kubernetes 1.33.
+
+## What is it?
+
+[Horizontal Pod autoscaling](/docs/tasks/run-application/horizontal-pod-autoscale/)
```

Comment:
I haven't seen this lower case "autoscaling" when other "Horizontal" and "Pod" are capitalised. Given this blog is going to talk about HPA more extensively, it may be more fitting to just use "Horizontal Pod Autoscaling"?